lizrd/scripts/run_exp_remotely.sh

```diff
@@ -34,5 +34,9 @@ run_grid_remotely() {
   ssh -t $host "$script"
 }
 
-# run your bash function
-run_grid_remotely $1 $2
+for i in "${@:2}"
```

Comment:
Can you add a comment what is happening here, as none of us is an expert in bash ^^